dotcom-rendering/src/components/marketing/lib/viewLog.ts

```diff
@@ -52,6 +48,5 @@ export const logEpicView = (testId: string): void => {
 		value: viewLog.slice(-maxLogEntries),
```

Comment:
We need to update `newValue` so it’s the actually an array, not an object containing the `value` key containing the array.

```ts
const newValue = viewLog.slice(-maxLogEntries);
storage.local.set(viewLogKey, newValue);
```